bin/create-scripts/block.js

```diff
@@ -0,0 +1,59 @@
+/** @format */
+const path = require( 'path' );
+const webpack = require( 'webpack' );
+
+const __rootDir = path.resolve( __dirname, '../../' );
+const entryPath = path.resolve( process.argv[ 2 ] );
+const sourceDir = path.dirname( entryPath );
+const outputDir = path.join( sourceDir, 'build' );
+const blockName = path.basename( path.dirname( entryPath ) );
+
+const baseConfig = require( path.join( __rootDir, 'webpack.config.js' ) );
```

Comment:
Do you reckon our block builder would only ever rely on Calypso Webpack config or Webpack config from whatever project the builder is used in, e.g. Jetpack?

Reply:
My 2 cents: There should be a shared webpack config that's part of a shared `calypso-sdk` npm or something like that.

Reply:
I'll second what @ockham said yeah - my vision is to have Calypso be the root and the thing and a new abstraction be the point of contact, not webpack

Reply:
Totally agreed, nobody else the underpinnings of `calypso-sdk` should know webpack exists.